common/CMakeLists.txt

```diff
@@ -6,6 +6,18 @@ set (ASEBACOMMON_SRC
 	msg/msg.cpp
 	msg/NodesManager.cpp
 )
+
+find_path(DNSSD_INCLUDE_DIR dns_sd.h)
```

Comment:
There is a missing search for the library. I understand that on OS X it is linked by default, but checking the header without specifying the library will likely break compilation on Linux and Windows.

Ah, if only we had a CI system that would try to merge and compile all pull requests ;-)